.github/ISSUE_TEMPLATE/dac_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: DAC request
+about: Request support for new DACs
+title: ''
+labels: DAC
+assignees: ''
+
+---
+
```

Comment:
@tmigone is DAC pretty common? We know it cuz we stare at it all day. Just thinking about the first-timer or someone who stumbles upon this page first. Maybe one quick sentence about DAC HATs to intro ? Let me know if you think otherwise.

Reply:
New DAC requests is the biggest single contributor in terms of issues :P
Most of the times new GitHub issues are not needed in the first place so I hope this redirects traffic to wher it's appropriate (forums or the master DAC list issue linked there).

BTW users will only see this when clicking "New issue" button.